relaton/models/people.py

```diff
@@ -7,35 +7,49 @@
 from .contacts import ContactMethod
 
 
-__all__ = ('Person', 'PersonName', 'PersonAffiliation', )
+__all__ = ('Forename', 'Person', 'FullName', 'GivenName', 'PersonAffiliation', )
 
 
 @dataclass
-class PersonName:
-    """Describes a person’s name."""
+class Forename(GenericStringValue):
+    """A forename of a person"""
 
-    completename: Optional[GenericStringValue] = None
-    """Full name.
-    Expected to be mutually exclusive with other properties.
-    """
+    initial: Optional[str] = None
```

Comment:
Can we add a docstring here and explain how it differs from `formatted_initials`?